apps/monarch/monarch-backend/src/workflows/postPendingPractitioner.ts

```diff
@@ -0,0 +1,49 @@
+import { Practitioner, PractitionerInfo } from "@c4c/monarch/common";
+import { Request } from "express";
+
+async function postNewPendingPractitioner(req: Request, postPendingPractitioner: (webhookData: Omit<PractitionerInfo, 'uuid'>) => Promise<Practitioner>) {
+    try {
+        const extractedPractitionerInfo = getPractitionerInfo(JSON.parse(req.body.rawRequest));
+        return postPendingPractitioner(extractedPractitionerInfo);
+    } catch (e) {
+        console.log(e);
+        throw new Error("Unable to post pending practitioner");
+    }
+}
+
+// List of field names for JotForm submission request objects
+const FIELD_NAMES = {
+  PHONE: 'q4_contact_number',
+  WEBSITE: 'q26_website',
+  MODALITY: 'q24_modality',
+  BUSINESS_LOCATION: 'q6_business_location',
+  BUSINESS_NAME: 'q32_business_name',
+  MIN_AGE_SERVED: 'q30_min_age_served',
+  EMAIL: 'q5_email',
+  FULL_NAME: 'q3_full_name',
+  LANGUAGES: 'q29_languages'
+}
```

Comment:
These are not the final form values - we'll have to work with LLB to get the the field names from their application form